controllers/auth.js

```diff
@@ -70,12 +70,21 @@ const updatePassword = async (req, res) => {
   res.sendStatus(204)
 }
 
+const unsetFirstLogin = async (req, res) => {
```

Comment:
I think we don't need an additional endpoint here. We can check it on the login endpoint if the user has flag { firstLogin: true }, on successful login, change it to false. I am waiting for your opinion.

Reply:
Agreed, will fix

Reply:
fixed